cmd/umserver/main.go

```diff
@@ -43,8 +45,21 @@ func main() {
 	}()
 	log.Printf("Server Listening at %s...", srv.Addr)
 
-	// TODO: There will be actual information about PostgreSQL connection in future
-	// ...
+	// pgConfig will be taken from package config, but it hasn't ready yet
+	pgConfig := storage.DBConfig{
+		Host:     "127.0.0.1",
+		Port:     "5432",
+		User:     "postgres",
+		Password: "postgres",
+		DBName:   "um_db",
+	}
+
+	db, err := storage.ConnectToDB(&pgConfig)
+	if err != nil {
+		log.Print(err)
+	}
+	defer db.Close()
```

Comment:
`db` can be `nil`

Reply:
Unfortunately I can't understand what your comment means. Couldn't describe in more detail. Thank you